### ISSUE TITLE
Missing Date Modified on dll causing issues with Nuget

### ISSUE THREAD

**Issue by kinjiru** [sibartlett/Geo#47]
When attempting to include Geo.dll in a nuget package the packaging fails with error 'The DateTimeOffset specified cannot be converted into a Zip file timestamp' because Geo.dll's Modified date is empty.

I believe nuget pack calls the CreateEntryFromFile method documented below, which sets the file's date in the nuget package to the file's Modified date:
https://docs.microsoft.com/en-us/dotnet/api/system.io.compression.zipfileextensions.createentryfromfile?view=netframework-4.7.2

As a workaround our CI process performs this step to set the Modified date before packaging:
`gci -path "geo.dll" -rec -file *.dll | Where-Object {$_.LastWriteTime -lt (Get-Date).AddYears(-20)} | %  { try { $_.LastWriteTime = '01/01/2020 00:00:00' } catch {} }`

Proposed solution:
Update Geo.dll build process to set the file's Modified date (same as Created date?)

![image](https://user-images.githubusercontent.com/2270183/81871282-d5b3af80-95ba-11ea-9005-c3ff55bd5aef.png)

**Comment by sibartlett** [comment]
I don't think this is an issue with recent tooling, so I am closing as I thinks a fix is no longer needed.
